acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -166,7 +168,7 @@ public void waitForNewFinalization() {
         MINUTES);
   }
 
-  public void waitForFullSyncCommitteeAggregate() {
+  public void waitForFullSyncCommitteeAggregate(final double percentFullRequirement) {
```

Comment:
I'm not sure it makes sense for `waitForFullSyncCommitteeAggregate` to have an option that lets it accept something less than full.  Probably either need to change the name or remove the param (ideally remove the param since a two node system really should be able to get perfect participation but may not be able to do that given that libp2p filters seen messages globally, not per topic).